src/engine/source/builder/builders/opBuilderLogParser.cpp

```diff
@@ -35,7 +35,7 @@ Builder getOpBuilderLogParser(std::shared_ptr<hlp::logpar::Logpar> logpar,
         if (!jsonDefinition.isArray())
         {
             throw std::runtime_error(fmt::format(
-                "Invalid json definition type: Expected \"array\" but got \"{}\"",
+                R"(Invalid json definition type: Expected "array" but got "{}")",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings